.github/workflows/release.yml

```diff
@@ -0,0 +1,94 @@
+name: Release
+
+on: [workflow_dispatch]
+
+jobs:
+  test:
+    name: "Test on Python ${{ matrix.python-version }} (${{ matrix.os }})"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ["3.9", "3.10", "3.11", "3.12"]
+    steps:
+      - uses: "actions/checkout@v4"
+        with:
+          fetch-depth: 0
+
+      - uses: "actions/setup-python@v5"
+        with:
+            python-version: "${{ matrix.python-version }}"
+
+      - name: "Installs for ${{ matrix.python-version }}"
+        run: |
+          pip install --upgrade pip
+          pip install nox
+
+      - name: "Run nox for Python ${{ matrix.python-version }}"
+        run: "nox -s test-${{ matrix.python-version }}"
+
+  build:
+    name: Build wheel and sdist
+    needs: test
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.12'
+      
+      - name: Install build dependency
+        run: |
+          pip install build
+          pip install --upgrade pip
+
+      - name: Build sdist and wheel
+        run: python -m build --wheel --sdist
```

Comment:
change run command to `python -m flit build` (this will bundle everything tracked by git) and automatically builds both sdist and wheels.

Reply:
`build` reads the `pyproject.toml`, detects flit as the backend and uses flit to build the wheel and sdist